packages/renderless/src/calendar-view/index.ts

```
@@ -160,6 +176,36 @@
   return res
 }
 
+const hasNoTime = (date) => {
+  const datetimeStr = date
+  let hoursTime = 0
+  let minutesTime = 0
+  let secondsTime = 0
+  const [datePart, timePart] = datetimeStr.split(' ')
+  const [year, month, day] = datePart && datePart.split('-')
+  if (timePart) {
+    const [hours, minutes, seconds] = timePart && timePart.split(':')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)